packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md

```diff
@@ -296,6 +302,106 @@ Returning an error object will cause Okta to return an OAuth 2.0 error to the re
   ]
 }
 ```
+### Sample Response to Replace an Existing Claim
+
+You can modify existing custom claims or OIDC standard profile claims, such as `birthdate` and `locale`. You can't, however, modify any system-specific claims, such as `iss` or `ver`, and you can't modify a claim that isn't currently part of the token in the request payload. Attempting to modify a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events.
+
+For the list of access token reserved claims that you can't modify, see [Access Tokens Scopes and Claims](https://developer.okta.com/docs/api/resources/oidc/#access-token-scopes-and-claims). Note that although the `aud` claim is listed as a reserved claim, you can modify that claim in a response.
+
```

Comment:
Note that although the `aud` and `sub` claim is listed as a reserved claim, you can modify that claim in access tokens.

We need to call this out because in ID Tokens you cannot modify sub and aud